app/models/patient.rb

```diff
@@ -117,35 +117,9 @@ def latest_transfer
     where(purged: false)
       .where(pause_notifications: false)
       .where('patients.id = patients.responder_id')
-      .where.not('latest_assessment_at >= ?', Time.now.in_time_zone('Eastern Time (US & Canada)').beginning_of_day)
-      .or(
-        where(purged: false)
-          .where(pause_notifications: false)
-          .where('patients.id = patients.responder_id')
-          .where(latest_assessment_at: nil)
-      )
-      .distinct
-  }
-
-  # Patients who are eligible for reminders:
-  #   - not purged AND
-  #   - notifications not paused AND
-  #   - valid preferred contact method AND
-  #   - HoH or not in a household AND
-  #   - we haven't sent them an assessment within the past 12 hours AND
-  #   - they haven't completed an assessment today OR they haven't completed an assessment at all
-  #   - (TODO in this scope rather than send_assessment) actively monitored OR has dependents that are being actively monitored
-  #
-  # NOTE: This method is currently being tested to be swapped in as the new reminder_eligible scope.
-  #       Once swapped in, the send_assessment method can be cut down to remove redundant logic.
-  scope :optimal_reminder_eligible, lambda {
-    where(purged: false)
-      .where(pause_notifications: false)
       .where.not(preferred_contact_method: ['Unknown', 'Opt-out', '', nil])
-      .where('patients.id = patients.responder_id')
       .where('last_assessment_reminder_sent <= ? OR last_assessment_reminder_sent IS NULL', 12.hours.ago)
-      .where('latest_assessment_at < ? OR latest_assessment_at IS NULL', Time.now.getlocal('-04:00').beginning_of_day)
-      .distinct
+      .where('latest_assessment_at < ? OR latest_assessment_at IS NULL', Time.now.in_time_zone('Eastern Time (US & Canada)').beginning_of_day)
   }
```

Comment:
Would you feel comfortable adding in any of these? These are statements pulled from the SA-770 scope that don't rely on the time zone attribute.

```ruby
monitoring_open
```

```ruby
.where(
        'patients.isolation = ? '\
        'OR patients.continuous_exposure = ? '\
        'OR patients.last_date_of_exposure >= ? '\
        'OR (patients.last_date_of_exposure IS NULL AND patients.created_at >= ?)',
        true,
        true,
        ADMIN_OPTIONS['monitoring_period_days'].days.ago.beginning_of_day,
        ADMIN_OPTIONS['monitoring_period_days'].days.ago.beginning_of_day
      )
```

```ruby
.or(
        # This OR is checking if the HoH has any dependents that make them eligible to
        # recieve notifications on the dependent's behalf.
        # The joined table is referred to as `dependents_patients` and is used for all
        # checks made on the dependents. Anywhere the `patients` table is specified or
        # where a hash is used for a condition are checks made on the HoH.
        joins(:dependents)
          .where(purged: false)
          .where(head_of_household: true)
          .where('patients.id = patients.responder_id')
          .where(
            # Ignore any joined rows where the dependents_patients is the HoH itself.
            'dependents_patients.id != dependents_patients.responder_id'
          )
          .where.not(
            # HoH is unconditionally ineligible if it has any of these preferred contact methods
            preferred_contact_method: ['Unknown', 'Opt-out', '', nil]
          )
          .where(
            # HoH is unconditionally ineligible if it has paused notifications
            pause_notifications: false
          )
          .where('dependents_patients.monitoring = ?', true)
          .where('dependents_patients.purged = ?', false)
          .where(
            # This is basically the same as active_dependents()
            # but we cannot use it here because it's a method, and even if it was a scope
            # it wouldnt be using the join table, 'dependents_patients'.
            'dependents_patients.isolation = ? '\
            'OR dependents_patients.continuous_exposure = ? '\
            'OR dependents_patients.last_date_of_exposure >= ? '\
            'OR (dependents_patients.last_date_of_exposure IS NULL AND dependents_patients.created_at >= ?)',
            true,
            true,
            ADMIN_OPTIONS['monitoring_period_days'].days.ago.beginning_of_day,
            ADMIN_OPTIONS['monitoring_period_days'].days.ago.beginning_of_day
          )
          .within_preferred_contact_time
      )
```

Reply:
Unfortunately no, because `monitoring_open` is not accurate for HoH (since they can be closed and still need to receive notifications because of dependents), and the other two are more than I feel comfortable changing in an emergency release without more testing and eyes.

Reply:
ok that sounds fine to me